pinot-common/src/main/java/com/linkedin/pinot/common/data/Schema.java

```diff
@@ -121,6 +130,19 @@ public void setMetricFieldSpecs(@Nonnull List<MetricFieldSpec> metricFieldSpecs)
     }
   }
 
+  @Nonnull
+  public List<DateTimeFieldSpec> getDateTimeFieldSpecs() {
+    return _dateTimeFieldSpecs;
+  }
+
+  public void setDateTimeFieldSpecs(@Nonnull List<DateTimeFieldSpec> dateTimeFieldSpecs) {
```

Comment:
re-checking whether this setter is needed. Same comment as before in terms of annotation and javadoc comments.

Reply:
I tried to make it consistent with the already existing metricFieldSpecs and dimensionFieldSpecs and timeFieldSpec. Those have setters, with no annotations. Should I add the annotations to those as well then?